sentinelhub/__init__.py

```diff
@@ -54,7 +54,7 @@
 )
 from .data_collections import DataCollection
 from .data_collections_bands import Band, Unit
-from .data_utils import get_failed_statistical_requests, statistical_to_dataframe
+from .data_utils import get_failed_statistical_requests
```

Comment:
was this one left in intentionally? I'd perhaps leave it out as well for now, and if it sees a lot of use we can maybe add it in later. It's much harder to remove it later (because importing from sentinelhub.data_utils will remain valid even if we add it to sentinelhub)

Reply:
Ok, I thought it was not using pandas so I left it in. I'll remove it from `__init__.py`